### PR TITLE
Aruha 2607 - rework docker image creation

### PR DESCRIPTION
**Pull request by antban**
Aruha 2607 - rework docker image creation

### CODE REVIEW

**Comment by antban** [comment]
:+1:

**Comment by adyach** [comment]
👍

File: Dockerfile [hunk 1]
```diff
@@ -1,37 +1,33 @@
-FROM registry.opensource.zalan.do/stups/python:3.5-cd28
+FROM registry.opensource.zalan.do/stups/openjdk:1.8.0-222-22
 MAINTAINER Team Aruha, team-aruha@zalando.de
 
-ENV KAFKA_VERSION="1.1.1" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.3.3"
+ENV KAFKA_VERSION="2.3.0" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.3.3"
```
Comment (adyach) [inline Dockerfile]: `1.6.2` vs `1.3.3`

File: Dockerfile [hunk 2]
```diff
@@ -1,37 +1,33 @@
...
 
-RUN mkdir -p "${SRC_PATH}"
-WORKDIR "${SRC_PATH}"
 ADD ./bubuku "${SRC_PATH}/bubuku"
 ADD ./requirements.txt "${SRC_PATH}/"
 ADD ./setup.py "${SRC_PATH}/"
-RUN pip3 install --no-cache-dir -r requirements.txt
+RUN cd "${SRC_PATH}" && pip3 install --no-cache-dir -r "requirements.txt" && python3 setup.py develop
 
 EXPOSE 9092 8080 8778 8888
```
Comment (adyach) [inline Dockerfile]: why do we need `8888` ?

File: delivery.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+version: "2017-09-20"
+pipeline:
+  - id: build
+    type: script
+    commands:
+      - desc: Install Docker
+        cmd: |
+          curl -fLOsS https://delivery.cloud.zalando.com/utils/ensure-docker && sh ensure-docker && rm ensure-docker
+
+      - desc: Build and push Docker image
```
Comment (adyach) [inline delivery.yaml]: do you think it makes sense to run tests from cdp? I see that travis file exists in the repo, which runs the tests.

File: Dockerfile (hunk 2 above)
Comment (antban) [reply Dockerfile]: This is default bubuku health port

File: Dockerfile (hunk 1 above)
Comment (antban) [reply Dockerfile]: It's nice that we want to update all the dependencies while merging repo. Nothing unpredictable could happen.

File: delivery.yaml (hunk 3 above)
Comment (antban) [reply delivery.yaml]: True, makes sense

File: Dockerfile (hunk 2 above)
Comment (adyach) [reply Dockerfile]: `ENV HEALTH_PORT=8080`
https://github.com/zalando-nakadi/bubuku/pull/144/files#diff-3254677a7917c6c01f55212f86c57fbfR7
what about that then ?

**Comment by antban** [comment]
:+1:

File: Dockerfile (hunk 2 above)
Comment (antban) [reply Dockerfile]: @adyach Removed it from config

**Comment by antban** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Pull request merged**